### ISSUE TITLE
LTE 

### ISSUE THREAD

**Issue by kawthar** [4gsim/4Gsim#2]
Hello,
thank you for sharing your work.
can you help me? when i run this project i find the follow error:

<!> Error: Attempt to load the oppsim shared library more than once. This usually happens when part of your simulation is using release libraries while other parts are using the debug version. Make sure to rebuild all parts of your model in either release or debug mode!

and with this project we can transmit the video streaming over LTE??

thank you for your help.

**Comment by 4gsim** [comment]
No, at the moment it only simulates the attach control part. Not sure about the error, did you get the master branch or radio, because master is working, radio is under construction?

**Comment by kawthar** [comment]
thank you for reply. I'm working with master branch.

**Comment by 4gsim** [comment]
looks like your problem is related to omnetpp. please do the following (recompile omnetpp with some special reconfiguration):
In Makefile.inc.in find the following lines (towards the end):

CMDENV_LIBS = -u _cmdenv_lib -loppcmdenv$D -loppenvir$D  
TKENV_LIBS = -u _tkenv_lib -lopptkenv$D -loppenvir$D -lopplayout$D                                    

and replace them with:
CMDENV_LIBS = -u _cmdenv_lib -Wl,--no-as-needed -loppcmdenv$D -loppenvir$D  
TKENV_LIBS = -u _tkenv_lib -Wl,--no-as-needed -lopptkenv$D -loppenvir$D -lopplayout$D                                    

then
./configure
make clean
make
